README.md

```diff
@@ -1123,7 +1123,8 @@ The following compilers are currently used in continuous integration at [AppVeyo
 | GCC 9.3.0 (Ubuntu 9.3.0-17ubuntu1~20.04)                                                               | Ubuntu 20.04.3 LTS | GitHub Actions |
 | GCC 10.2.0 (Ubuntu 10.2.0-5ubuntu1~20.04)                                                              | Ubuntu 20.04.3 LTS | GitHub Actions |
 | GCC 11.1.0                                                                                             | Ubuntu (aarch64)   | Drone CI       |
-| GCC 12.0.0 20211219 (experimental)                                                                     | Ubuntu 20.04.3 LTS | GitHub Actions |
```

Comment:
Why drop GCC 12?

Reply:
This is always the latest snapshot build and GCC 12 hasn't made it into the r-toolchain PPA yet (https://launchpad.net/~ubuntu-toolchain-r/+archive/ubuntu/test).

Reply:
Exactly. It would be about time to have multiple images to install gcc 12 just from a recent Ubuntu, yet then I would lose the older Clangs…

Reply:
Makes sense, thanks.